webapp/crashstats/settings/bundles.py

```diff
@@ -34,12 +34,15 @@
     "@fortawesome/fontawesome-free": ["css/all.min.css", "webfonts/*"],
     "tablesorter": ["dist/css/theme.default.min.css", "dist/js/jquery.tablesorter.js"],
     "d3": ["dist/*"],
-    "jssha": ["dist/*.js"],
+    "jssha": ["dist/*.js", "dist/*.map"],
     "qs": ["dist/*"],
     "moment": ["moment.js"],
     "filesize": ["lib/*"],
     "underscore": ["*.js"],
-    "jquery-jsonview": ["dist/*"],
+    "jquery.json-viewer": [
+        "json-viewer/jquery.json-viewer.css",
+        "json-viewer/jquery.json-viewer.js",
+    ],
```

Comment:
jquery-jsonview has a file that indicates there's a `.map` file just like jssha did, but there is no `.map` file in the package. The library hasn't been touched in 8 years. So I switched us to jquery.json-viewer which has some minor differences, but it's been updated more recently and doesn't have the `.map` problem.